semantics.json

```diff
@@ -54,7 +54,9 @@
       "ol",
       "h2",
       "h3",
-      "hr"
+      "hr",
+      "pre",
+      "code
```

Comment:
@icc I'll add the missing double quote for you :-)

Reply:
Thanks! :sweat_smile: